scene-recognition/readme.md

```diff
@@ -75,8 +75,8 @@ The subtypes of slates (blue) is also important to annotate.
 However, the non-important cases (grey) are various different negative cases that are not frames of interest. These may be similar to positive cases. These are sometimes less distinct between each other. Do the best possible, but move on if too much time is spent figuring out the distinctions.  
 Add the [modifier](https://docs.google.com/document/d/1IyM_rCsCr_1XQ39j36WMX-XnVVBT4T_01j-M0eYqyDs/edit#heading=h.xnfilznsrhpe) where needed. I.e. Pick the most preferred, clearest `type label`, add "Shift" when making the key combo. 
 
-### How to Annotate It
-The tool uses one or two key-combination presses to annotate the different kinds of frames. A key combination can be a single key, or could be a combo like "Shift P". Press the relevant one to annotate the `type`. 
+### How to Annotate It (Annotators)
```

Comment:
Is there any reason for this line to be plural and line51 to be singular? 

There is a minor typo that doesn't hurt readability in line47"need"-> "needs".

Reply:
I guess this was also a typo.

Reply:
I've removed those additions to the section headers, the preparation step makes its clear the Pm's work is to do the preparation. The rest is annotator work as usual.